ballerina/annotation.bal

```diff
@@ -26,3 +26,10 @@ public type ServiceDescriptorData record {|
 
 # Service descriptor annotation.
 public annotation ServiceDescriptorData ServiceDescriptor on service;
+
+
+public type AuthServiceConfig record {|
+    ListenerAuthConfig[] auth?;
+|};
+
+public annotation AuthServiceConfig AuthConfig on service;
```

Comment:
Shall we update the annotation as follows in order to match with other modules (HTTP, WebSocket and GraphQL)?
@dilanSachi @BuddhiWathsala @daneshk WDYT?
```suggestion
public type GrpcServiceConfig record {|
    ListenerAuthConfig[] auth?;
|};

public annotation GrpcServiceConfig ServiceConfig on service;
```
Please add the documentation also.

Reply:
+1 to match with other modules

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/343/commits/c156f03ddbc865d79ce4d3d3faf6183ef8016204